utils/deepcopy.go

```diff
@@ -1,11 +0,0 @@
-package utils
-
-// Simplistic map copy
-func copyMap(src map[string]interface{}) map[string]interface{} {
-	dst := make(map[string]interface{}, len(src))
-
-	for k, v := range src {
-		dst[k] = v
-	}
-	return dst
-}
```

Comment:
was this util func just unused?

Reply:
Yep, at least golangci-lint flagged it as unused and I couldn't find any reference/usage to this function